include/z64save.h

```diff
@@ -7,13 +7,15 @@
 typedef struct {
     /* 0x00 */ u8 buttonItems[4];
     /* 0x04 */ u8 cButtonSlots[3];
-    /* 0x08 */ u16 equipment;
+    /* 0x08 */ u16 equipment; // a mask where each half-byte corresponds to a type of equipment `EquipmentType`, and
+                              // each half-byte is a piece `EquipValue*`
 } ItemEquips; // size = 0x0A
 
 typedef struct {
     /* 0x00 */ u8 items[24];
     /* 0x18 */ s8 ammo[16];
-    /* 0x28 */ u16 equipment;
+    /* 0x28 */ u16 equipment; // a mask where each half-byte corresponds to a type of equipment `EquipmentType`, and
+                              // each bit to an owned piece `EquipInv*`
```

Comment:
same discussion re line breaks in structs, dunno if we'll be able to come to an agreement soon though. wont let it hold up the pr if so